src/Infrastructure/Persistence/Configurations/TodoItemConfiguration.cs

```diff
@@ -1,18 +1,15 @@
 ﻿using CleanArchitecture.Domain.Entities;
-using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
 namespace CleanArchitecture.Infrastructure.Persistence.Configurations
 {
-    public class TodoItemConfiguration : IEntityTypeConfiguration<TodoItem>
+    public class TodoItemConfiguration : BaseConfiguration<TodoItem>
     {
-        public void Configure(EntityTypeBuilder<TodoItem> builder)
+        public override void Configure(EntityTypeBuilder<TodoItem> builder)
         {
             builder.Ignore(e => e.DomainEvents);
```

Comment:
This configuration wouldn't be required, except for DomainEvents. It might be good to move domain events to a BaseEntity. Something to consider.